videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
+            for videoframe in self.videoframes:
```

Comment:
Also; according to [this](http://stackoverflow.com/questions/7688725/multiple-actions-in-list-comprehension-python) it seems like the convention is to not use list comprehensions with commands :)